docs: update A2A agent card to 0.3 schema (#6051)
mintlify/docs: ai/skillmd.mdx

| `protocolVersion`     | A2A protocol version. Always `0.3`.                                                                          |
| `preferredTransport`  | Default transport for clients that do not negotiate. Always `HTTP+JSON`.                                     |
| `supportedInterfaces` | Array of `{ url, protocolBinding, protocolVersion }` entries describing how agents can reach your site.      |
| `provider`            | `{ url, organization }` identifying the documentation site. `organization` is the site title.                |
| `defaultInputModes`   | Media types the agent accepts as input. Always `["text/plain"]`.                                             |
| `defaultOutputModes`  | Media types the agent produces. Always `["text/plain"]`.                                                     |
| `capabilities`        | Feature flags. Currently `{ streaming: false, pushNotifications: false }`.                                   |
| `skills`              | The skills exposed at `/.well-known/agent-skills/`.                                                          |

> URLs in the card (`url`, `documentationUrl`, `provider.url`, and each skill URL) are built from your configured custom domain when one is set, so the published card always advertises the canonical domain instead of the `*.mintlify.app` subdomain.

Reviewer: ```suggestion
URLs in the card (`url`, `documentationUrl`, `provider.url`, and each skill URL) use your configured custom domain, so the published card always advertises the canonical domain instead of the `*.mintlify.app` subdomain.
```
